src/main/resources/migrations/00079_add_agreement_permission.sql

```diff
@@ -0,0 +1,13 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:agreements_read_update_permission
+INSERT INTO  permissions(id, name, description) VALUES ((SELECT MAX(id) + 1 FROM permissions), 'agreements:read', 'View agreements');
+INSERT INTO  permissions(id, name, description) VALUES ((SELECT MAX(id) + 1 FROM permissions), 'agreements:update', 'Update agreements');
+INSERT INTO  role_permission VALUES ((SELECT id FROM roles WHERE name = 'super-admin'), (SELECT id FROM permissions WHERE name = 'agreements:read'), NOW(), NOW());
+INSERT INTO  role_permission VALUES ((SELECT id FROM roles WHERE name = 'admin'), (SELECT id FROM permissions WHERE name = 'agreements:read'), NOW(), NOW());
+INSERT INTO  role_permission VALUES ((SELECT id FROM roles WHERE name = 'view-and-refund'), (SELECT id FROM permissions WHERE name = 'agreements:read'), NOW(), NOW());
+INSERT INTO  role_permission VALUES ((SELECT id FROM roles WHERE name = 'view-only'), (SELECT id FROM permissions WHERE name = 'agreements:read'), NOW(), NOW());
+
+INSERT INTO  role_permission VALUES ((SELECT id FROM roles WHERE name = 'super-admin'), (SELECT id FROM permissions WHERE name = 'agreements:update'), NOW(), NOW());
+INSERT INTO  role_permission VALUES ((SELECT id FROM roles WHERE name = 'admin'), (SELECT id FROM permissions WHERE name = 'agreements:update'), NOW(), NOW());
+INSERT INTO  role_permission VALUES ((SELECT id FROM roles WHERE name = 'view-and-refund'), (SELECT id FROM permissions WHERE name = 'agreements:update'), NOW(), NOW());
```

Comment:
Do we want view and refund users to be able to modify agreements?

Reply:
In that case, I think not granting the permission to view and refund users (leaving it as the preserve of admins) would be best so we do not have to take permissions away from people in the future. Can’t put the genie back in the bottle and all that. 🧞

Reply:
Yeah agree @alexbishop1, have pushed to https://github.com/alphagov/pay-adminusers/commit/a512fc4218a4524b83870e6cf69be9e4835c29ec